Core/Generators/CSharp/CSharpGenerator.cs

```diff
@@ -46,13 +46,18 @@ public override ValueTask<string> Compile(BebopSchema schema, GeneratorConfig co
                 builder.Indent(indentStep).AppendLine();
             }
 
-            if (Schema.Definitions.Values.Any(d => d is ConstDefinition))
+            if (Config.EmitBinarySchema || Schema.Definitions.Values.Any(d => d is ConstDefinition))
             {
                 builder.AppendLine(ConstantSummary);
                 builder.AppendLine(GeneratedAttribute);
                 builder.AppendLine($"public static class BopConstants {{");
                 builder.Indent(indentStep);
 
+                if (Config.EmitBinarySchema)
+                {
+                    builder.AppendLine(Schema.ToBinary().ConvertToString("public static readonly byte[] BEBOP_SCHEMA = new byte[] {", "};"));
```

Comment:
This needs to be public static read only BebopSchema = new byte[] to compile. Failing tests.

Reply:
Tests passed. It was `const` before.